dev/devicelab/bin/tasks/large_image_changer_perf.dart

```diff
@@ -0,0 +1,40 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter_devicelab/framework/adb.dart';
+import 'package:flutter_devicelab/framework/framework.dart';
+import 'package:flutter_devicelab/framework/utils.dart';
+import 'package:flutter_devicelab/tasks/perf_tests.dart';
+
+const String kPackageName = 'com.example.macrobenchmarks';
+const String kActivityName = 'com.example.macrobenchmarks.MainActivity';
+
+class LargeImageChangerPerfTest extends MemoryTest {
```

Comment:
I wonder if more memory info (e.g., peak memory, average, percentile, etc.) from `DevToolsMemoryTest` (see, e.g., [this example][1]) or Timeline (see, e.g., [profiling_summarizer][2] and https://github.com/flutter/engine/pull/18516) would be helpful here? The DevTools works for Android and the Timeline works for iOS.

[1]: https://github.com/flutter/flutter/blob/master/dev/devicelab/bin/tasks/complex_layout_scroll_perf__devtools_memory.dart

[2]: https://github.com/flutter/flutter/blob/master/packages/flutter_driver/lib/src/driver/profiling_summarizer.dart#L79